src/sigstore.ts

```diff
@@ -43,21 +49,27 @@ type IdentityProviderOptions = Pick<
   'identityToken' | 'oidcIssuer' | 'oidcClientID' | 'oidcClientSecret'
 >;
 
+function newTLogClient(options: { rekorBaseURL?: string }): TLog {
```

Comment:
prefixing with `new` feels weird in js, I initially thought it was a typo and should be `new TLogClient` on line 63. 

Thoughts on just down casing the instance?
```suggestion
function tLogClient(options: { rekorBaseURL?: string }): TLog {
```

Reply:
oh wait it's returning a "new tlog client".. maybe
```suggestion
function createTLogClient(options: { rekorBaseURL?: string }): TLog {
```

Reply:
👍